### PR TITLE
Quick update for the 2 new danish pull-request...

### PR DESCRIPTION
**Pull request by BenjaminHoegh**
Quick update for the 2 new danish pull-request there adds Danish phone number and postal codes

Add translate for:
 - phoneDK
 - postalcodeDK

### CODE REVIEW

File: src/localization/messages_da.js [hunk 1]
```diff
@@ -30,5 +30,7 @@ $.extend( $.validator.messages, {
 	maxWords:  $.validator.format( "Indtast venligst højst {0} ord" ),
 	step: $.validator.format( "Angiv en værdi gange {0}." ),
 	notEqualTo: "Angiv en anden værdi, værdierne må ikke være det samme.",
-	integer: "Angiv et ikke-decimaltal, der er positivt eller negativt."
+	integer: "Angiv et ikke-decimaltal, der er positivt eller negativt.",
```
Comment (HawkonDK) [inline src/localization/messages_da.js]: don't know why GitHub say this are re-added

Comment (CRC-Mismatch) [reply src/localization/messages_da.js]: I guess it should detect and show that the only change was the comma at the end, but instead for some reason it got parsed as an entire block together with the next 2 lines...

**Comment by staabm** [comment]
Needs rebase after we merged #2071 #2070

**Comment by BenjaminHoegh** [comment]
Moved to #2165 because of GitHub issues

**Pull request closed without merging**